Description: Correct pod NAME issues in LedgerSMB/DBObject/Menu.pm
 Has an incomplete module name and a colon instead of dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472567&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Menu.pm
+++ b/LedgerSMB/DBObject/Menu.pm
@@ -1,6 +1,6 @@
 =head1 NAME
 
-LedgerSMB::Menu:  Menu Handling Back-end Routines for LedgerSMB
+LedgerSMB::DBObject::Menu - Menu Handling Back-end Routines for LedgerSMB
 
 =head1 SYNOPSIS
 
